src/components/headerVideo.tsx

```diff
@@ -0,0 +1,46 @@
+// ----- Imports ----- //
+
+import React, { FC } from 'react';
+import { css } from '@emotion/core'
+import { remSpace, neutral } from '@guardian/src-foundations';
+
+import { darkModeCss } from 'styles';
+import { Option } from 'types/option';
+import { Video } from 'video';
+
+// ----- Component ----- //
+
+const styles = css`
+    margin: 0 0 ${remSpace[2]} 0;
+    position: relative;
+    display: block;
+    width: 100%;
+    padding-bottom: 56.25%;
```

Comment:
Is this because all videos will be the same aspect ratio?

Reply:
yes they are in the templates.